specification/use_case_library.md

```diff
@@ -5,42 +5,42 @@ The following table contains a set of commonly performed FinOps scenarios that w
-| FinOps Practitioner | Data Analysis and Showback | As a FinOps practitioner, I need to analyze service costs month over month, over a time period | EffectiveCost<br>BillingPeriodStart<br>Provider<br>ServiceName |
-| FinOps Practitioner | Data Analysis and Showback | As a FinOps practitioner, I need to analyze service costs, by region, over a time period | EffectiveCost<br>BillingPeriodStart<br>Provider<br>Region<br>ServiceName |
-| FinOps Practitioner | Data Analysis and Showback | As a FinOps practitioner, I need to analyze Compute Engine service costs month over month for a period of time to identify accounts spending the most money on Compute Engine | BilledCost<br>BillingPeriodStart<br>Provider<br>ResourceId<br>ResourceName<br>ServiceName<br>SubAccountId<br>SubAccountName |
-| FinOps Practitioner | Data Analysis and Showback | As a FinOps practitioner, I want to monitor how much we are spending on a specific SaaS product purchased via the cloud service provider's marketplace. | ChargePeriodStart<br>ChargePeriodEnd<br>EffectiveCost<br>InvoiceIssuer<br>Provider<br>Publisher |
-| FinOps Practitioner | Data Analysis and Showback | As a FinOps Practitioner, I need to understand what we are spending across providers, billing periods, and service categories | Provider<br>BillingPeriodStart<br>BilledCost<br>BillingCurrency<br>ServiceCategory |
+| Finance | Data Analysis and Showback | As a Finance person of a company that sells SaaS services, I need to determine the resource quantity and type used by a customer so that a monthly invoice can be issued to the customer. | ProviderName <br>BillingPeriodStart <br>SKU<br>PricingQuantity<br>ConsumedQuantity<br>UsageUnit <br>Tags |
+| Finance | Data Analysis and Showback | As a person in Finance, I need a report of all cost associated with a product from all geographic locations for a given month. | BilledCost<br>BilledCurrency<br>BillingAccountId<br>BillingAccountName<br>BillingPeriodEnd<br>ProviderName<br>Tags |
+| Finance | FinOps & Intersecting Frameworks | As a person in Finance, I need a report of service-level cost within a specific Sub Account as a part of a private pricing negotiation. | BillingPeriodStart<br>EffectiveCost<br>ProviderName<br>ServiceName<br>SubAccountID<br>SubAccountName |
+| Finance | Forecasting | As a person in Finance, I need to forecast amortized costs on a month over month basis, based on historical trends | BillingPeriodStart<br>ChargeType<br>EffectiveCost<br>PricingUnit<br>ProviderName<br>ServiceType<br>ServiceCategory |
+| Finance | Forecasting | As a person in Finance, I need to forecast cashflow on a month over month basis, based on historical trends | BillingPeriodStart<br>ChargeType<br>ChargeDescription<br>BilledCost<br>PricingUnit<br>ProviderName<br>ServiceType<br>ServiceCategory |
```

Comment:
- `ServiceCategory` or `ServiceName  instead of `ServiceType`
- `PricingUnit` even though there is no `PricingQuantity`? should this be `BillingCurrency`?

Reply:
```suggestion
| Finance | Forecasting | As a person in Finance, I need to forecast cashflow on a month over month basis, based on historical trends | BillingPeriodStart<br> ChargeCategory<br>ChargeDescription<br>BilledCost<br> BillingCurrency <br>ProviderName<br> ServiceName<br>ServiceCategory |
```

Reply:
```suggestion
| Finance | Forecasting | As a person in Finance, I need to forecast cashflow on a month over month basis, based on historical trends | BillingPeriodStart<br>ChargeType<br>ChargeDescription<br>BilledCost<br> BillingCurrency <br>ProviderName<br>ServiceType<br>ServiceCategory |
```